loadProtocol.js

```diff
@@ -293,9 +293,14 @@ function eqs (compareTo, k) {
 }
 
 function loadProtocol (version) {
-  const data = require('minecraft-data')(version).protocol
-  const comments = require('minecraft-data')(version).protocolComments
-  $j('#protocolTable').html(protocolToString(data, comments))
+  if (version.startsWith('bedrock')) {
+    const [, v] = version.split('_')
+    $j('#protocolTable').html(`<iframe src="protocol/bedrock/${v}" frameborder="0" style='width: 100%; height: 80vh;'><a href="protocol/bedrock/${v}">Click here</a></iframe>`)
```

Comment:
Do we really need an iframe ?

Reply:
The html could be loaded into the page with jquery, however the styles used in the html file which is self contained could collide with the page styles so I used an iframe here.

Reply:
why can't we use browserify like the rest of this package ?

Reply:
ah it's this compiler thing from protodef yaml hm

Reply:
yeah well still... there has to be a better way than an iframe...

Reply:
I took a look at removing the header and putting in the css here, but the issue is that it uses bootstrap and without it it looks a lot uglier IMO.